pdc_client/__init__.py

```diff
@@ -203,10 +205,21 @@ def get_paged(self, res, **kwargs):
         """
```

Comment:
I have update the docstring, please help to review that, and thanks for pointing it.